config/23.1/ports.conf

```diff
@@ -201,6 +201,7 @@ security/strongswan
 security/stunnel				arm
 security/sudo
 security/suricata				arm,aarch64
+security/tailscale				arm,aarch64
```

Comment:
Why ignore arm and aarch64?

https://www.freshports.org/security/tailscale/ shows it works on both arches.

Reply:
usually just to not pollute build time for others... also because build has not been verified as we 99% deal with amd64